src/mjolnir/graphenhancer.cc

```diff
@@ -68,6 +68,7 @@ constexpr float kDensityLatDeg = (kDensityRadius * kMetersPerKm) / kMetersPerDeg
 constexpr float kTurnChannelFactor = 1.25f;
 constexpr float kRampDensityFactor = 0.8f;
 constexpr float kRampFactor = 0.85f;
```

Comment:
we discussed setting kRampFactor to 0.7. I don't think you need kRampDensityFactor